debian/matrix-synapse.service

```diff
@@ -6,8 +6,8 @@ Type=simple
 User=matrix-synapse
 WorkingDirectory=/var/lib/matrix-synapse
 EnvironmentFile=/etc/default/matrix-synapse
```

Comment:
(putting this comment here for want of anywhere else to put it):

we're currently installing the default file as `/etc/default/matrix-synapse-py3`, so this won't work. We shouldn't rename it, because that will drop any local changes that people have made.